pkg/proxy/proxy/transparent.go

```diff
@@ -175,6 +199,14 @@ func (t *TransparentProxyService) ServeMetadata(addr string) {
 	}
 }
 
+// Check that the requested url is allowed by the network policy.
+func (proxy TransparentProxyService) ApplyNetworkPolicy(req *http.Request, ctx *goproxy.ProxyCtx) (*http.Request, *http.Response) {
+	if proxy.Mode != EnforcementMode {
```

Comment:
I'm now realizing that "monitor"ing a policy as implemented is actually ignoring that policy. do you think it's worth having a dry-run / audit of the policy? If not, maybe  just tweak the wording to be something like "disabled" (and we could add in a "monitor" at a later point, if necessary).

Reply:
hmm okay so in that case another option could be to rename this to PolicyMode and have "disabled", "monitor/dry-run/audit", "enforce". If we want to add that middle option, i think prefer that but if we don't, i think i could work with the current naming scheme

Reply:
I've renamed it to PolicyMode and only added disabled/enforce for now and I'll create a new bug to add a dry-run mode.